ballerina/types.bal

```diff
@@ -25,7 +25,7 @@ public type ConnectionConfig record {|
     # The Salesforce endpoint URL
     string baseUrl;
     # Configurations related to client authentication
-    http:BearerTokenConfig|config:OAuth2RefreshTokenGrantConfig auth;
+    http:BearerTokenConfig|config:OAuth2RefreshTokenGrantConfig|config:OAuth2PasswordGrantConfig|config:OAuth2ClientCredentialsGrantConfig auth;
```

Comment:
Seems this line exceeds the 120 char limit, shall we fix this?

Reply:
updated in [8385ec2](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/302/commits/8385ec2ccf542c86b49f917b37d071f806bc0341)